src/facets/PantosRegistryFacet.sol

```diff
@@ -191,7 +196,7 @@ contract PantosRegistryFacet is IPantosRegistry, PantosBaseFacet {
      */
     function setMinimumServiceNodeDeposit(
```

Comment:
`setMinimumServiceNodeDeposit` is missing in roles doc. worth updating it.

Reply:
Should be done now